ocaml/xapi-cli-server/records.ml

```diff
@@ -5242,3 +5242,93 @@ let vtpm_record rpc session_id vtpm =
           ()
       ]
   }
+
+let tracing_record rpc session_id tracing =
+  let _ref = ref tracing in
+  let empty_record =
+    ToGet (fun () -> Client.Tracing.get_record ~rpc ~session_id ~self:!_ref)
+  in
+  let record = ref empty_record in
+  let x () = lzy_get record in
+  {
+    setref=
+      (fun r ->
+        _ref := r ;
+        record := empty_record
+      )
+  ; setrefrec=
+      (fun (a, b) ->
+        _ref := a ;
+        record := Got b
+      )
+  ; record= x
+  ; getref= (fun () -> !_ref)
+  ; fields=
+      [
+        make_field ~name:"uuid" ~get:(fun () -> (x ()).API.tracing_uuid) ()
+      ; make_field ~name:"hosts"
+          ~get:(fun () ->
+            map_and_concat get_uuid_from_ref (x ()).API.tracing_hosts
+          )
+          ~set:(fun s ->
+            let hosts =
+              get_words ',' s
+              |> List.map (fun uuid ->
+                     Client.Host.get_by_uuid ~rpc ~session_id ~uuid
+                 )
+            in
+            Client.Tracing.set_hosts ~rpc ~session_id ~self:tracing ~hosts
+          )
+          ()
+      ; make_field ~name:"name_label"
+          ~get:(fun () -> (x ()).API.tracing_name_label)
+          ()
+      ; make_field ~name:"tags"
+          ~get:(fun () -> Record_util.s2sm_to_string ";" (x ()).API.tracing_tags)
```

Comment:
In the rest of the module the separator for this function includes a space:
```suggestion
          ~get:(fun () -> Record_util.s2sm_to_string "; " (x ()).API.tracing_tags)
```

Reply:
Resolved by [ca7c05e](https://github.com/xapi-project/xen-api/pull/4966/commits/ca7c05e791c3a3d238ed4b11da845dffd118a126)